draft-ietf-quic-transport.md

```diff
@@ -73,9 +73,11 @@ informative:
     title: "Structured Streams: A New Transport Abstraction"
     author:
       - ins: B. Ford
-    date: 2007-08
+    date: 2007-10
```

Comment:
Yeah, if you cite it as "proceedings", you cite the conference date. If you cite it as published in CCR, you cite the issue publication date.